packages/gafl-webapp-service/src/lib/address-lookup-service.js

```diff
@@ -0,0 +1,47 @@
+import fetch from 'node-fetch'
+import { ADDRESS_LOOKUP_SERVICE } from '../constants.js'
+import db from 'debug'
+const debug = db('webapp:address-lookup-service')
+
+export default async (premises, postcode) => {
+  const url = new URL(process.env.ADDRESS_LOOKUP_URL)
+
+  const params = new URLSearchParams({
+    postcode: postcode,
+    premises: premises,
+    key: process.env.ADDRESS_LOOKUP_KEY,
+    lang: ADDRESS_LOOKUP_SERVICE.lang,
+    dataset: ADDRESS_LOOKUP_SERVICE.dataset
+  })
+
+  url.search = params.toString()
+
+  debug({ url })
+
+  const { results } = await (async () => {
+    try {
+      const response = await fetch(url.href, {
```

Comment:
Should be setting a timeout here to prevent the incoming request from timing out if the lookup service fails to respond in time.

Reply:
Yes actually good point - I think a 10 second timeout would be appropriate but I will add it as an environment variable